src/om_bootstrap/button.cljs

```diff
@@ -8,7 +8,7 @@
             [om-tools.dom :as d :include-macros true]
             [om-tools.mixin :refer-macros [defmixin]]
             [schema.core :as s])
-  (:require-macros [schema.macros :as sm]))
+  (:require-macros [schema.core :as sm]))
```

Comment:
We already include `schema.core :as s`. Can you just remove the `require-macros` line and change `sm` to `s`?